### PR TITLE
Feature/hero banner addons

### PR DESCRIPTION
**Pull request by raphaellueckl**
Hero Banner - Now with badge and area for other information.

### CODE REVIEW

File: src/components/30-organisms/commercial-hero-banner/README.md [hunk 1]
```diff
@@ -117,10 +117,126 @@ Import the commercial-hero-banner-defining script and use a commercial-hero-bann
   <h1 slot="title">...</h1> // Only one title
   <p slot="content">...</p> // You can put multiple <p> elements with the 'content' slot
   <small slot="disclaimer">...</small> // Only one disclaimer
-  <axa-button-link slot="button" /> // You can put multiple <p> elements with the 'content' slot
+  <axa-button-link slot="button" /> // Can also be 'axa-button', meant as CTA (Call to Action)
+  <div slot="badge">...</div> // Any DOM node, meant to visualize a badge displayed at the upper right position
```
Comment (LucaMele) [inline src/components/30-organisms/commercial-hero-banner/README.md]: Please use a more flexible name like: `top-right-section` and `bottom-right-section`

File: src/components/30-organisms/commercial-hero-banner/index.js [hunk 2]
```diff
@@ -82,9 +82,11 @@ class AXACommercialHeroBanner extends InlineStyles {
                       class="o-commercial-hero-banner__content-button"
```
Comment (LucaMele) [inline src/components/30-organisms/commercial-hero-banner/index.js]: Why buttons? better name would be `section-right`

**Comment by LucaMele** [comment]
Please reafactor as said.

Short: one wrapping section instead of 2. name of class buttons to something like `addon-section`

**Comment by raphaellueckl** [comment]
I addressed your issues @NoahLiechti and @LucaMele. Please have a look again.

**Comment by NoahLiechti** [comment]
Looks good!

**Pull request merged**